source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -15,8 +13,16 @@ Retrieve Data
 Overview
 --------
 
-In this guide, you can learn how to retrieve data from your MongoDB
-collections with the {+driver-long+}.
+You can perform find operations to retrieve data from your MongoDB database.
+You can perform a find operation to match documents on a set of criteria
+by calling the ``Find()`` method.
```

Comment:
Issue:

I think omitting the driver from the description makes the learning objectives for the page less clear. I also think that "database" is ambiguous since the pattern used the CRUD pages use the database/collection reference rather than the datastore/server reference.

Suggestion:

I think keeping "[by using] the {+driver-long+}" would be helpful. Since it seems like the intent is to introduce the Find() method prior to the admonition, perhaps the following structure could work as well as keep consistent:

"In this guide, you can learn how to retrieve documents from your MongoDB collections by using read operations. You can call the ``Find()`` method on a collection to retrieve documents that match a set of criteria."

Reply:
Question:

I wasn't able to find any changes to the other sections to ensure that the method name is mentioned prior to the admonition. If that was what was intended, I think it would be helpful to update those as well.